src/git.rs

```diff
@@ -155,6 +158,16 @@ impl RepoExt for git2::Repository {
         self.checkout_head(Some(git2::build::CheckoutBuilder::new().force()))?;
         Ok(Some(commit_id))
     }
+
+    fn gc(&self) -> error::Result<()> {
+        ::std::process::Command::new("git")
+            .args(&["gc"])
+            .current_dir(self.path())
+            .stderr(Stdio::null())
+            .stdout(Stdio::null())
+            .spawn()?.wait()?;
```

Comment:
Instead of `spawn()`, we should probably use `status()` and then check that the status is a success status. Right now, this is just checking that we spawned the subprocess and that it exited, it isn't verifying that the subprocess exited OK. Also we should import `Command` up where we import `Stdio`, and not be lazy like I was in the other file (that was just supposed to be a temporary thing so I cut corners).

```rust
let status = Command::new("git")
    .arg("gc")
    .current_dir(self.path())
    .stdout(Stdio::null())
    .stderr(Stdio::null())
    .status()?;
if !status.success() {
    return Err(error::Error::GitGcFailed); // You'll need to add this `Error` variant in src/error.rs
}
Ok(())
```